content/en/watchdog/insights.md

```diff
@@ -19,79 +19,132 @@ further_reading:
-2. Click **Kubernetes**.
-3. Select the **Explorer** tab at the top of the page.
-4. Choose one of the Kubernetes resource types in the **Select Resources** box.
-5. A list of your Kubernetes resources appears, with the Watchdog Insights panel at the top.
+**Error Outlier**
 
-#### Live Processes
+Error outliers display fields such as [faceted tags or attributes][3] that contain characteristics of errors that match the current search query. Statistically overrepresented `key:value` pairs among errors can provide hints into the root cause of issues. Typical examples of error outliers include `env:staging`, `version:1234`, and `browser.name:Chrome`.
 
-Watchdog Insights appear in [Live Processes][12].
+In the **banner card** view, you can see:
 
-1. In the left navigation, hover over **Infrastructure**.
-2. Click **Processes**.
+* The field name
+* The proportion of total errors and overall RUM events that the field contributes to
+* Related tags
 
-A list of your Processes and associated data appears, with the Watchdog Insights panel at the top. 
+In the **full side panel**, you can see a timeseries graph about the total number of RUM errors with the field along with a impact pie charts and a list of RUM events that contain the field.
 
-#### Serverless
+{{< img src="real_user_monitoring/explorer/watchdog_insights/error_outlier_side_panel-1.png" alt="Error Outlier full side panel" style="width:100%;" >}}
 
-For serverless infrastructures, Watchdog surfaces the following insights:
+**Latency Outlier**
+
+Latency outliers display fields such as [faceted tags or attributes][1] that are associated with performance bottlenecks that match the current search query. `key:value` pairs with worse performance than the baseline can provide hints into the performance bottlenecks among a subset of real users.
+
+Latency outliers are computed for [Core Web Vitals][2] such as First Contentful Paint, First Input Delay, Cumulative Layout Shift, and [Loading Time][3]. For more information, see [Monitoring Page Performance][2].
+
+In the **banner card** view, you can see:
+
+* The field name
+* The performance metric value containing the field and the baseline for the rest of the data
+
+In the **full side panel**, you can see a timeseries graph about the performance metric with an X axis of increments of `p50`, `p75`, `p99`, and `max`, along with a list of RUM events that contain the field. 
+
+{{< img src="real_user_monitoring/explorer/watchdog_insights/latency_outlier_side_panel-1.png" alt="Latency Outlier full side panel view" style="width:100%;" >}}
+
+[1]: /real_user_monitoring/explorer/search/#facets
+[2]: /real_user_monitoring/browser/monitoring_page_performance/#core-web-vitals
+[3]: /real_user_monitoring/browser/monitoring_page_performance/#monitoring-single-page-applications-spa
+{{% /tab %}}
+{{% tab "Serverless" %}}
+
+TODO
+
+For serverless infrastructures, Watchdog surfaces the following outliers:
```

Comment:
I still think those are not outliers per the definition at the beginning of the doc 
```Calculated on the product data matching the active query, it surfaces tags that either appear too frequently in some event types (e.g: errors) or drive some continuous metrics upwards (e.g: latency).```